lib/loader.js

```diff
@@ -90,7 +90,7 @@ module.exports = async function (content, sourceMap) {
       if (component.template.lang === 'pug') {
         const pug = requirePeer('pug')
         try {
-          component.template.content = pug.render(component.template.content)
+          component.template.content = pug.render(component.template.content, {filename: this.resourcePath})
```

Comment:
```suggestion
          component.template.content = pug.render(component.template.content, { filename: this.resourcePath })
```

Reply:
Enable this next time please
![image](https://user-images.githubusercontent.com/16421948/61879707-f41ac280-af36-11e9-9004-d40e1f7ab40e.png)

Reply:
I'll sure do that next time. (Was offline for a few days so didn't notice your comment)